lti_consumer/lti_1p3/consumer.py

```diff
@@ -475,16 +475,22 @@ def lti_ags(self):
     def enable_ags(
         self,
         lineitems_url,
+        lineitem_url,
+        ags_interaction_model='programmatic'
```

Comment:
I think we should go with `allow_programatic_grade_interaction` and use a boolean here. It's better than my original suggestion in the ticket.

Reply:
Updated